src/renderer/screens/exchange/Swap2/utils/shared/useSwapTransaction.js

```diff
@@ -77,15 +78,31 @@ export type SwapTransactionType = {
   reverseSwap: () => void,
 };
 
-const useSwapTransaction = (): SwapTransactionType => {
+const useSwapTransaction = ({
+  defaultCurrency,
+  defaultAccount,
+  defaultParentAccount,
+}: {
+  defaultCurrency?: $PropertyType<SwapSelectorStateType, "currency">,
+  defaultAccount?: $PropertyType<SwapSelectorStateType, "account">,
+  defaultParentAccount?: $PropertyType<SwapSelectorStateType, "parentAccount">,
+} = {}): SwapTransactionType => {
   const [toState, setToState] = useState<SwapSelectorStateType>(SelectorStateDefaultValues);
-  const [fromState, setFromState] = useState<SwapSelectorStateType>(SelectorStateDefaultValues);
+  const [fromState, setFromState] = useState<SwapSelectorStateType>({
+    ...SelectorStateDefaultValues,
```

Comment:
Lowercase, it's not a class nor a component

Reply:
You mean for `SelectorStateDefaultValues` ? I just reused the var here, but yeah I can find/replace everywhere.

Reply:
👍🏼